src/main/java/adeuxpas/back/entity/Ad.java

```diff
@@ -0,0 +1,184 @@
+package adeuxpas.back.entity;
+
+import adeuxpas.back.enums.AdStatus;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+
+@Entity
+public class Ad {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @NotNull
+    @Column(length = 150)
+    private String title;
+    @NotNull
+    @Column(name = "article_description", columnDefinition = "TEXT")
+    private String articleDescription;
+    @NotNull
+    @Size(max = 150, message = "title must be at most 150 characters long")
```

Comment:
I don't understand why the @ Size annotation is necessary here. Could you explain it to me?

Reply:
it actually has nothing to do here , I remove it